AmazonChimeSDK/AmazonChimeSDK/internal/video/DefaultVideoClientController.swift

```diff
@@ -229,6 +229,34 @@ extension DefaultVideoClientController: VideoClientDelegate {
             }
         }
     }
+    
+    public func remoteVideoSourcesDidBecomeAvailable(_ sourcesInternal: [RemoteVideoSourceInternal]) {
+        var sources = [RemoteVideoSource]()
```

Comment:
should you check for empty? We don't want to give callback when `sourcesInternal` is empty.